### PR TITLE
Integrate unicore-hub SSO functionality

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony Hey, I have this here. What am I doing right, wrong, and where can I go next? :+1:

File: mc2/templates/password_reset_done.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; {% trans 'Password reset' %}
+</div>
+{% endblock %}
+
+{% block title %}{{ title }}{% endblock %}
+{% block content_title %}<h1>{{ title }}</h1>{% endblock %}
+{% block content %}
```
Comment (miltontony) [inline mc2/templates/password_reset_done.html]: @codiebeulaine use `{% block body %}` here instead of `content`

**Comment by justinvdm** [comment]
@codiebeulaine reviewed the password reset bits of the PR, looks good, left some comments.

**Comment by miltontony** [comment]
:+1:  :cake: all around!!

**Pull request merged**